### ISSUE TITLE
OEP Data Upload Wizard and API: Clarify table & column name syntax 

### ISSUE THREAD

**Issue by amanda-wein** [OpenEnergyPlatform/oeplatform#1826]
## Description of the issue

When creating a new table in the upload wizard, the only guidance given for table and column names is that lower case letters, numbers, and underscores are the only valid characters, and that the character limit is 50. Based on this, a column name that is simply a single number, or a column name that starts with a number and is followed by letters or underscores, should be accepted. However, if one tries to create a table with this column name, the column name field is highlighted to indicate an error and the error message "Invalid definitions" is displayed after clicking the Create Table button. However, the tooltip guidance indicates that this column name is valid.

![colname-1](https://github.com/user-attachments/assets/c361d7fe-333c-462d-8c4e-8cbd98ad5e96)

This is a restriction of PostgreSQL, that column names can't start with numbers. That isn't mentioned in the data upload wizard, or in any documentation for the wizard or the API. If one does try to use an invalid column name, the error messages (in both the wizard and the API) indicate that something has gone wrong, but don't give the user any information as to what the problem is or how to fix it.

## Ideas of solution

The following adjustments would make the data upload process clearer:
- In the data upload wizard, edit the tooltip guidance to indicate that column names cannot start with a number.
- In the data upload wizard documentation, add details about the correct formatting of table and column names.
- In the API documentation, add details about the correct formatting of table and column names.
- In the data upload wizard, make the error messages more detailed to indicate to the user exactly where & what the error is if a table can't be created successfully.
- In the API, make the error messages more detailed to indicate to the user exactly where & what the error is if a table can't be created successfully.

## Workflow checklist

- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)

**Comment by Ludee** [comment]
That is correct, there is a PostfreSQL constraint that column names cannot start with a number.
This should be added to the tooltip.
Same applies to table names as well.
